README.md

```diff
@@ -130,7 +130,7 @@ It can:
 The curator's role is to curate the vault, meaning setting risk limits, gates, allocators, fees.
 Only one address can have this role.
 
-Some actions of the curator are timelockable (between 0 and 3 weeks, or infinite if the action has been frozen).
+Curator actions that can safely be timelocked are timelockable.
```

Comment:
actually I don't like it anymore because some things need to be timelocked in order to get the vault's properties 
```suggestion
Curator actions are timelockable, except the ones that cannot safely be.
```